src/reducers/toyReducer.js

```diff
@@ -1,35 +1,25 @@
 export const initialState = {
-  toys: []
+  toys: [],
```

Comment:
You'll want to watch for consistencies here (I haven't checked-- they may all be-- just calling it out).  So, if you decide you want to go with trailing commas in this object, you should make sure you leave trailing commas on ALL objects across your codebase.  

I personally am not a fan of trailing commas, but this is purely personal preference-- you can decide which you want to use, but then make sure you keep it consistent across the whole app.

Reply:
Prettier adds the trailing commas, I can edit it off of my preferences? If I make sure that I have prettier implemented throughout my app then it should be consistent, right?

Reply:
That's fine, you can leave the Prettier defaults.  Just be aware there is a way to configure Prettier to use whatever rules you wish to use.  For example, our team has specific rules that we'll enforce across the team, so you'd just want to grab someone's Prettier config from the team so you're all using the same configuration.

Reply:
Oh okay, that makes sense because you want it to be similar code throughout all applications.